logging/cluster-logging-release-notes.adoc

```diff
@@ -11,6 +11,9 @@ include::snippets/logging-compatibility-snip.adoc[]
 
 include::snippets/logging-stable-updates-snip.adoc[]
 
+:context: cluster-logging-rn
```

Comment:
I'm not sure if these context bits are required since I think the context is already set at the top of the assemblies?
e.g. `:context: cluster-logging-release-notes-v5x`
I haven't seen this done elsewhere so it might be, but I just haven't used it this way before.

Reply:
https://redhat-documentation.github.io/modular-docs/#reusing-modules